pydarn/plotting/fan.py

```diff
@@ -0,0 +1,246 @@
+
+        # Time for coordinate conversion
+        dtime = dt.datetime(dmap_data[plot_beams[0][0]]['time.yr'],
+                            dmap_data[plot_beams[0][0]]['time.mo'], dmap_data[plot_beams[0][0]]['time.dy'],
+                            dmap_data[plot_beams[0][0]]['time.hr'], dmap_data[plot_beams[0][0]]['time.mt'],
+                            dmap_data[plot_beams[0][0]]['time.sc'])
+
+        # Get radar beam/gate locations
+        beam_corners_aacgm_lats, beam_corners_aacgm_lons=radar_fov(dmap_data[0]['stid'],
+            coords='aacgm', date=dtime)
+        fan_shape = beam_corners_aacgm_lons.shape
+
+        # Work out shift due in MLT
+        beam_corners_mlts = np.zeros((fan_shape[0], fan_shape[1]))
+        mltshift = beam_corners_aacgm_lons[0, 0] - \
+            (aacgmv2.convert_mlt(beam_corners_aacgm_lons[0, 0], dtime) * 15)
+        beam_corners_mlts = beam_corners_aacgm_lons - mltshift
+
+        # Hold the beam positions
+        thetas = np.radians(beam_corners_mlts)
+        rs = beam_corners_aacgm_lats
+
+        # Get range-gate data and groundscatter array for given scan
+        scan = np.zeros((fan_shape[0] - 1, fan_shape[1]-1))
+        grndsct = np.zeros((fan_shape[0] - 1, fan_shape[1]-1)) #initialise arrays
+        for i in np.nditer(plot_beams):
+            try:
+                slist = dmap_data[i.astype(int)]['slist'] #get a list of gates where there is data
+                beam = dmap_data[i.astype(int)]['bmnum'] #get the beam number for the record
+                scan[slist, beam] = dmap_data[i.astype(int)][parameter]
+                grndsct[slist, beam] = dmap_data[i.astype(int)]['gflg']
+            # if there is no slist field this means partial record
+            except KeyError:
+                continue
+
+        # Colour table and max value selection depending on parameter plotted
+        # Load defaults if none given
+        # TODO: use cmaps as over writting cmap is bad practice...
+        # did I do that in my code ... hmm
+        if cmap is None:
+            cmap = {'p_l': 'plasma', 'v': PyDARNColormaps.PYDARN_VELOCITY,
+                    'w_l': PyDARNColormaps.PYDARN_VIRIDIS,
+                    'elv': PyDARNColormaps.PYDARN}
+            cmap = plt.cm.get_cmap(cmap[parameter])
+
+        # Setting zmin and zmax
+        defaultzminmax = {'p_l': [0, 50], 'v': [-200, 200],
+                          'w_l': [0, 250], 'elv': [0, 50]}
+        if zmin is None:
+            zmin = defaultzminmax[parameter][0]
+        if zmax is None:
+            zmax = defaultzminmax[parameter][1]
+
+        # Setup plot
+        # This may screw up references
+        if ax is None:
+            ax = plt.axes(polar=True)
+            if beam_corners_aacgm_lats[0,0] > 0:
+                ax.set_ylim(90, lowlat)
+                ax.set_yticks(np.arange(lowlat, 90, 10))
+            else:
+                ax.set_ylim(-90, -lowlat)
+                ax.set_yticks(np.arange(-lowlat, -90, -10))
```

Comment:
```suggestion
                ax.set_ylim(-90, lowlat)
                ax.set_yticks(np.arange(lowlat, -90, -10))
```
I believe this will fix @Devin-Huyghebaert bug found when setting a lowlat value for the southern hemisphere. In this case you now would have to input a -ve value, which I guess makes more sense than it changing the sign for you. I've done it as a suggestion just incase anyone wants to argue for the previous case?

Reply:
By 'I believe' I should say, it does fix it, I have tested it

Reply:
Actually this has an issue with default now, bear with me.

Reply:
```suggestion
                ax.set_ylim(-90, -abs(lowlat))
                ax.set_yticks(np.arange(-abs(lowlat), -90, -10))
```
This is the only solution I can think up so that you can have a positive number default relate to the southern hemisphere. This way you can put -30 in and it'll do -30, but if you put 30 in it'll change it to -30. The caveat being you wont be able to plot past 0degrees on a southern hemisphere plot, which I can't see being a problem anyway as it would look ridiculous on a polar plot? As long as we're not going to move to make spherical plots with northern *and* southern fov on I think this will probably suffice?